packages/tc-markdown-parser/lib/property-coercers/index.js

```diff
@@ -37,40 +37,43 @@ module.exports = {
 	  about `hasMany` is `String` and `Subdocument` types, which in our case
 	  includes Codes.
 	*/
-	String(subdocument, { hasMany = false } = {}) {
+	String(subdocument) {
 		const items = split(subdocument);
 
-		// Expecting a list, got a list
-		if (hasMany && items.length) {
-			return {
-				valid: true,
-				value: items.map(flattenNodeToPlainString),
-			};
-		}
-
-		// Expecting a list, didn't get a list'
-		if (hasMany && !items.length) {
+		// Not expecting a list, but got a list
+		if (items.length) {
 			return {
 				valid: false,
-				value: "expected a list, but didn't get any bulleted items",
+				value: 'expected a single item, but got a bulleted list',
 			};
 		}
 
-		// Not expecting a list, didn't get a list'
-		if (!hasMany && !items.length) {
+		return {
+			valid: true,
+			value: flattenNodeToPlainString(subdocument),
+		};
```

Comment:
No longer we don't need to consider `hasMany` in string coercer because nested definition will be processed at NestedString coercer